### PR TITLE
[google_cloud_texttospeech] Fixed for 2.0.0 and later versions.

### PR DESCRIPTION
**Pull request by iory**
# What is this

In `google-cloud-texttospeech`, the submodules enums and types have been removed from v2.0.0.

(ref: https://github.com/googleapis/python-texttospeech/blob/master/UPGRADING.md#enums-and-types )
With this PR, I've made some changes that will work with 2.0.0 and later versions.

### CODE REVIEW

File: 3rdparty/google_cloud_texttospeech/bin/text2wave [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+    from google.cloud.texttospeech import SsmlVoiceGender
+    from google.cloud.texttospeech import SynthesisInput
+    from google.cloud.texttospeech import VoiceSelectionParams
+else:
+    import google.cloud.texttospeech
+    AudioEncoding = google.cloud.texttospeech.enums.AudioEncoding
+    SsmlVoiceGender = google.cloud.texttospeech.enums.SsmlVoiceGender
+    AudioConfig = google.cloud.texttospeech.types.AudioConfig
+    SynthesisInput = google.cloud.texttospeech.types.SynthesisInput
+    VoiceSelectionParams = google.cloud.texttospeech.types.VoiceSelectionParams
```
Comment (knorth55) [inline 3rdparty/google_cloud_texttospeech/bin/text2wave:27]: can't we write like this?

```suggestion
    from google.cloud.texttospeech.enums import AudioEncoding
    from google.cloud.texttospeech.enums import SsmlVoiceGender
    from google.cloud.texttospeech.types import AudioConfig
    from google.cloud.texttospeech.types import SynthesisInput
    from google.cloud.texttospeech.types import VoiceSelectionParams
```

Comment (iory) [reply 3rdparty/google_cloud_texttospeech/bin/text2wave]: Thanks. I modified it. Please check it.

File: 3rdparty/google_cloud_texttospeech/setup.cfg [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[flake8]
+exclude = .?*, src/, build, docs
+
+[isort]
+force_single_line = true
+force_sort_within_sections = true
+lexicographical = true
+order_by_type = false
+lines_after_imports = 2
+skip_glob = *__init__.py
```
Comment (knorth55) [inline 3rdparty/google_cloud_texttospeech/setup.cfg:10]: do we need `setup.cfg`?

Comment (iory) [reply 3rdparty/google_cloud_texttospeech/setup.cfg:10]: Sorry, miss commit. I'm rebasing.

**Comment by iory** [comment]
Rebased. 
Could you review this PR?

**Review by knorth55 — APPROVED** [review]

**Comment by iory** [comment]
Thanks!

**Comment by k-okada** [comment]
closed via #274

**Pull request closed without merging**